circle/docker-release-image.sh

```diff
@@ -285,6 +285,24 @@ chart_update_image() {
   esac
 }
 
+chart_update_appVersion() {
```

Comment:
Perhaps the simplest way is to just use an env var to skip the appVersion update in some repos. So, we could disable this in [parse-dashboard](https://github.com/bitnami/bitnami-docker-parse-dashboard) but enable it in [parse](https://github.com/bitnami/bitnami-docker-parse), that way we're only following the parse appVersion. The downside is that it's difficult to get visibility into what appVersion is being followed and it is easy to configure incorrectly and have multiple components updating the appVersion.